src/com/lesson_4_3/MainTest.java

```diff
@@ -0,0 +1,97 @@
+import org.openqa.selenium.NoAlertPresentException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+//import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MainTest {
+    WebDriver driver;
+
+    @org.junit.Before
+    public void setUp() throws Exception {
+        System.setProperty("webdriver.chrome.driver", "chromedriver.exe");
+        driver = new ChromeDriver();
+        driver.get("https://the-internet.herokuapp.com/javascript_alerts");
+    }
+
+    @org.junit.After
+    public void tearDown() throws Exception {
+        driver.close();
+    }
+
+    @Test
+    public void alertTest() {
+        WebElement button = driver.findElement(By.xpath("//*[contains(text(), 'Click for JS Alert')]"));
+        button.click();
+        try{
+            Alert alert = driver.switchTo().alert();
+            String textOnAlert = alert.getText();
+            assertEquals("I am a JS Alert",textOnAlert);
+            alert.accept();
+            WebElement result = driver.findElement(By.xpath("//p[@id='result']"));
+            String a = result.getText();
+            assertEquals("You successfuly clicked an alert",a);
+        }
+        catch (NoAlertPresentException e)
+        {
+            e.printStackTrace();
+        }
+
+    }
+    @Test
+    public void confirmTest() {
+        WebElement button = driver.findElement(By.xpath("//*[contains(text(), 'Click for JS Confirm')]"));
+        button.click();
+        try{
+            Alert alert = driver.switchTo().alert();
+            String textOnAlert = alert.getText();
+            assertEquals("I am a JS Confirm",textOnAlert);
+            alert.accept();
+            WebElement result = driver.findElement(By.xpath("//p[@id='result']"));
+            String a = result.getText();
+            assertEquals("You clicked: Ok",a);
+        }
+        catch (NoAlertPresentException e)
+        {
+            e.printStackTrace();
+        }
+    }
+    @Test
+    public void promtTest() {
+        WebElement button = driver.findElement(By.xpath("//*[contains(text(), 'Click for JS Prompt')]"));
+        button.click();
+        try{
+            Alert alert = driver.switchTo().alert();
+            String textOnAlert = alert.getText();
+            assertEquals("I am a JS prompt",textOnAlert);
+            alert.sendKeys("Test");
+            alert.accept();
+            WebElement result = driver.findElement(By.xpath("//p[@id='result']"));
+            String a = result.getText();
+            assertEquals("You entered: Test",a);
+        }
+        catch (NoAlertPresentException e)
+        {
+            e.printStackTrace();
+        }
+    }
```

Comment:
All 3 tests will pass in case if alert even doesn't appear after click on buttons (you can easily miss bugs). You have several options to make your tests correct:
- do not put all code into try section, but just code, which can throw exception;
- throw RuntimeException in catch section;
- throw your custom exception in catch section.

Options 1 and 3 is preferable for me.